src/wallet/wallet.h

```diff
@@ -256,8 +256,8 @@ class CKeyPool
 
 /** A wrapper to reserve an address from a wallet
  *
- * ReserveDestination is used to reserve an address. It is passed around
- * during the CreateTransaction/CommitTransaction procedure.
+ * ReserveDestination is used to reserve an address.
+ * It is currently only used inside of CreateTransaction.
```

Comment:
Not quite true, it is still used in GetNewChangeDestination() which is called by the `getrawchangeaddress` RPC